src/doc/rustc/src/platform-support/wasm32-unknown-emscripten.md

```diff
@@ -0,0 +1,98 @@
+# `wasm32-unknown-emscipten`
+
+**Tier: 2**
+
+The `wasm32-unknown-emscripten` target is a WebAssembly compilation target which
+uses the Emscripten compiler toolchain. Unlike the
+[`wasm32-unknown-unknown`](./wasm32-unknown-unknown.md) target, which only has
+partial support for the Rust standard library, `wasm32-unknown-emscripten`
+integrates with JavaScipt on the web to provide a more featureful standard library.
+This target is often used when compiling for the web, accessing web APIs, or
+interoperating with C and C++ code.
+
+One existing user of this target (please feel free to edit and expand this list
+too) is the [`pyodide` project](https://github.com/pyodide/pyodide) which provides
```

Comment:
Not sure if the `please feel free to edit and expand this list too` part was intentional but it can probably be removed.

Reply:
Done